### PR TITLE
Add 1.7.0-alpha02 to changelog

### PR DESCRIPTION
**Pull request by mazunin-v-jb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,67 @@
+# 1.7.0-alpha02 (July 2024)
+_Changes since 1.7.0-alpha01_
+
+## Breaking Changes
+
+### Multiple Platforms
+- [ComposeScene `fun hitTestInteropView(position: Offset): Boolean` changed to `fun hitTestInteropView(position: Offset): InteropView?`](https://github.com/JetBrains/compose-multiplatform-core/pull/1426)
```
Comment (MatkovIvan) [inline CHANGELOG.md:7]: `ComposeScene` is internal API and shouldn't be mentioned here. Especially as "breaking change"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,67 @@
+# 1.7.0-alpha02 (July 2024)
+_Changes since 1.7.0-alpha01_
```
Comment (MatkovIvan) [inline CHANGELOG.md:2]: [`MD022` - Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md022.md)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+### Resources
+- [_(prerelease fix)_ Fix an android app compose resources packaging broken after introduction AS previews](https://github.com/JetBrains/compose-multiplatform/pull/5090)
+- [Now drawables from upper DPIs will be downscalled to the expected size. (the same behavior as on Android)](https://github.com/JetBrains/compose-multiplatform/pull/5101)
+
+### Gradle plugin
+- [_(prerelease fix)_ Fix "InvalidUserDataException: Cannot change hierarchy of dependency configuration" on Gradle sync](https://github.com/JetBrains/compose-multiplatform/pull/5076)
+
+## Dependencies
+- Gradle Plugin `org.jetbrains.compose`, version `1.7.0-alpha02`. Based on Jetpack Compose libraries:
+  - Compiler 1.5.14.1-beta02
```
Comment (MatkovIvan) [inline CHANGELOG.md:53]: There no such version of "Jetpack Compose libraries"

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,67 @@
...
+  - [Runtime 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-runtime#1.7.0-beta05)
+  - [UI 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.7.0-beta05)
+  - [Foundation 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.7.0-beta05)
+  - [Material 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-material#1.7.0-beta05)
+  - [Material3 1.3.0-beta03](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.3.0-beta03)
```
Comment (MatkovIvan) [inline CHANGELOG.md:58]: @igordmn are we going to merge beta04 before alpha02?

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (angelix) [reply CHANGELOG.md:58]: @igordmn @MatkovIvan why not beta05?

Comment (MatkovIvan) [reply CHANGELOG.md:58]: @angelix next time, alpha02 is already later than we wanted. beta04 was the last version when we did actual merge (https://github.com/JetBrains/compose-multiplatform-core/pull/1461)

Comment (igordmn) [reply CHANGELOG.md:54]: > @igordmn are we going to merge beta04 before alpha02?

No, there is no [changes](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.3.0-beta04) in code there

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,63 @@
...
+### Resources
+- [_(prerelease fix)_ Fix an android app compose resources packaging broken after introduction AS previews](https://github.com/JetBrains/compose-multiplatform/pull/5090)
+- [Now drawables from upper DPIs will be downscalled to the expected size. (the same behavior as on Android)](https://github.com/JetBrains/compose-multiplatform/pull/5101)
+
+### Gradle plugin
+- [_(prerelease fix)_ Fix "InvalidUserDataException: Cannot change hierarchy of dependency configuration" on Gradle sync](https://github.com/JetBrains/compose-multiplatform/pull/5076)
+
+## Dependencies
+- Gradle Plugin `org.jetbrains.compose`, version `1.7.0-alpha02`. Based on Jetpack Compose libraries:
+- - [Compiler 1.5.14](https://developer.android.com/jetpack/androidx/releases/compose-compiler#1.5.14)
```
Comment (igordmn) [inline CHANGELOG.md:49]: Let's remove this. Compiler is moved to Kotlin 2, and this version is only for Kotlin 1

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,62 @@
...
+- [Interop views are now correctly clipped when their measured clipped and unclipped bounding boxes don't match](https://github.com/JetBrains/compose-multiplatform-core/pull/1430)
+- [Touches inside interop views are not exclusive to them and are processed on Compose side as well.](https://github.com/JetBrains/compose-multiplatform-core/pull/1426)
+- [Fix `material3.ModalBottomSheet` safe area usage](https://github.com/JetBrains/compose-multiplatform-core/pull/1438)
+- [Fix hiding interop element during quick scroll](https://github.com/JetBrains/compose-multiplatform-core/pull/1425)
+- [_(prerelease fix)_ Fixed floating cursor isn't working](https://github.com/JetBrains/compose-multiplatform-core/pull/1443)
+- [Fixed the keyboard appearing when selecting from SelectionContainer](https://github.com/JetBrains/compose-multiplatform-core/pull/1448)
+- [Fix status bar padding on iPad devices](https://github.com/JetBrains/compose-multiplatform-core/pull/1442)
+- [VoiceOver doesn't allow to perform a11y actions (scrolling, activate, customActions) when disabled() semantics is present in affected element](https://github.com/JetBrains/compose-multiplatform-core/pull/1446)
+
+### Desktop
```
Comment (MatkovIvan) [inline CHANGELOG.md:36]: Please add https://github.com/JetBrains/compose-multiplatform-core/pull/1467

**Review by MatkovIvan — APPROVED** [review]

**Review by igordmn — APPROVED** [review]

**Pull request merged**